analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/LocksShouldBeReleased.Mutex.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
+using System.Threading;
+
+// https://docs.microsoft.com/en-us/dotnet/standard/threading/mutexes
+internal class Foo
+{
+    bool cond;
+    public Mutex instanceMutex;
+    public static Mutex staticMutex;
+
+    // Note that dispose does not release the resource
```

Comment:
It releases resource, but other one (`waitHandle`).
I'd also move this comment around line 38  where the dispose is happening
```suggestion
    // Note that dispose does not release the mutex
```